draft-ietf-ppm-dap.md

```diff
@@ -809,6 +809,11 @@ A task's parameters are immutable for the lifetime of that task. The only way to
 change parameters or to rotate secret values like collector HPKE configuration
 or the VDAF verification key is to configure a new task.
 
+This document does not specify a mechanism for configuring each party with the
+task parameters. Such a mechanism SHOULD be transparent so that each party has
+an opportunity to opt out based on the value of any parameter. (See
```

Comment:
Reviewer note: I considered refining this to "the value of any _privacy-relevant_ parameter", but I thought it would be a bit bike-sheddy to work out which parameters are privacy-relevant and which aren't.